kubectl-plugin/pkg/cmd/job/job_submit.go

```diff
@@ -104,6 +104,9 @@ func NewJobSubmitCommand(streams genericclioptions.IOStreams) *cobra.Command {
 		Example: jobSubmitExample,
 		RunE: func(cmd *cobra.Command, args []string) error {
 			entryPointStart := cmd.ArgsLenAtDash()
+			if entryPointStart == -1 || len(args[entryPointStart:]) == 0 {
```

Comment:
Is it trivial to add a test for this case?

Reply:
IMO, yes, because I think we if we add a test for this case, I would be like testing the `ArgsLenAtDash` function.